srcpkgs/flowblade/template

```diff
@@ -14,4 +14,6 @@ maintainer="shizonic <realtiaz@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://jliljebl.github.io/flowblade/"
 distfiles="https://github.com/jliljebl/flowblade/archive/v${version}.tar.gz"
-checksum=3753363d15caa6f1901118132a87f92d3b5e843c325775dbd83b30ee4d2a9bbf
+checksum=9431dc80a3ff3a899953cd5864c00335b355fb44c0601895b0dd57268b40bdf9
+# Tests fail indicating that the test modules cannot be imported
+make_check=extended
```

Comment:
if they fail, then they should be disabled, not just marked as extended

Reply:
I really appreciate your guidance. Even though I read the documentation on how to use the `make_check` variable, I wasn't sure if it was advisable to leave the option disabled. Since the checks worked with the `make_check=extended` option, that's why I pushed the pull requests.

I have already made the changes you suggested.

Reply:
CI and -Q don't run extended tests, that's the only reason why it works, if you run xbps-src with -K (which also runs extended tests), it will fail. Please disable tests that are not working, there is no point in marking broken tests as "extended".